### PR TITLE
Test multi-dc in a separate project

### PR DESCRIPTION
**Pull request by chbatey**
* Test multi-dc works for akak 2.5.6 keeping main dep at 2.4
* Upgrade to sbt 0.13.16
* Upgrade to akka 2.4.20

0.4 was broken as it compared nodes from different DCs, this test will
fail if that is re-introduced

### CODE REVIEW

File: cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      val dcBSystem = ActorSystem("MultiDcSystem", config.withFallback(dcB))
+      implicit val materialiser = ActorMaterializer()
+
+      try {
+        val cluster = Cluster(dcASystem)
+        ClusterHttpManagement(cluster).start().futureValue
+        val response = Http().singleRequest(HttpRequest(uri = s"http://127.0.0.1:$httpPortA/members")).futureValue
+        response.status should equal(StatusCodes.OK)
+        val members = Unmarshal(response.entity).to[ClusterMembers].futureValue
+        members.members.size should equal(2)
```
Comment (chbatey) [inline cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala]: I could test more, before this it blew up entirely

Comment (patriknw) [reply cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala]: so it was already fixed in master?

Comment (chbatey) [reply cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala]: Yep, by accident by the looks of it: https://github.com/akka/akka-management/commit/e4fc9d5849342590daba3495597f68bd68887f9c

Comment (raboof) [reply cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala]: If that's the fix are we sure it's sufficient? It looks like it'd still blow up when members from different dc's are in the status `MemberStatus.Up` (https://github.com/akka/akka-management/commit/e4fc9d5849342590daba3495597f68bd68887f9c#diff-c6868e05e6924399e82f52ce16d1a10cR73)

Comment (raboof) [reply cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala]: Ah sorry already fixed further in https://github.com/akka/akka-management/pull/52/files#diff-c6868e05e6924399e82f52ce16d1a10cL74

**Pull request merged**